src/PlugWallet/index.ts

```diff
@@ -221,16 +193,10 @@ class PlugWallet {
         Principal.fromText(to),
         parseInt(token.index.toString(), 10)
       );
-      // Optimistically update the state
-      const collections = this.collections.map(col => ({
-        ...col,
-        tokens: col.tokens.filter(tok => tok.index !== token.index),
-      }));
-      this.collections = collections.filter(col => col.tokens.length);
       getCachedUserNFTs({ userPID: this.principal, refresh: true }).catch(
```

Comment:
what's `getCachedUserNFTs` used for?

Reply:
Requests nfts from kyasshu's cache layer